languages/haskell/src/DMN/Translate/FEELhelpers.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE LambdaCase #-}
+
+module DMN.Translate.FEELhelpers where
+
+import Data.Char
+import Data.List
+import DMN.Types
+
+wrapArray :: String -> [String] -> String
+wrapArray myop xs = "[" ++ intercalate myop xs ++ "]"
+
+showFeels ch fexps = "\"" ++ varname ch ++ "\":" ++ if squash
+                                                    then showFeel $ head fexps
+                                                    else wrapArray "," (showFeel <$> fexps)
+  where squash = maybe True (\case
+                                DMN_List _ -> False
+                                _          -> True) (vartype ch)
+          
+showFeel :: FEELexp -> String
+showFeel (FNullary (VS str))  = show str
+showFeel (FNullary (VN num))  = show num
+showFeel (FNullary (VB bool)) = toLower <$> show bool
+showFeel (FSection Feq  (VB rhs)) = "(x)=>x === "++ (toLower <$> show rhs)
```

Comment:
the `FSection` cases here are intended to produce syntax for lambda functions.

In JS and TS, lambda functions are defined compactly as `(x)=>……`.

The Python version is a bit more verbose, a bit like `lambda x: ……`

This section of the code hasn't really been exercised until now – I have added an output column to the `language/haskell/README.md` that illustrates the return of lambda functions.